Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -177,16 +203,23 @@ public class LeapServiceProvider : LeapProvider {
       }
 #endif
 
+      if (_updateHandInPrecull!=_prevUpdateHandInPrecull && !_updateHandInPrecull) {
```

Comment:
You could simplify this logic to !_updateHandInPrecull && _prevUpdateHandInPrecull